jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -107,6 +108,22 @@ public void setEnvironment(Map<String, String> environment) {
     this.environment = environment;
   }
 
+  @Input
+  @Nullable
```

Comment:
Remove `@Nullable`. I hope NullAway got this.

Reply:
I've been having problems running builds locally that I haven't looked into.